Source/Core/Core/Boot/Boot.h

```diff
@@ -162,3 +170,5 @@ void UpdateStateFlags(std::function<void(StateFlags*)> update_function);
 /// Normally, this is automatically done by ES when the System Menu is installed,
 /// but we cannot rely on this because we don't require any system titles to be installed.
 void CreateSystemMenuTitleDirs();
+
+void RAMOverrideForIPLMemoryValues(std::string& data, const u32 ipl_hash);
```

Comment:
This doesn't need to be exposed. This function can be an internal static function. It just needs to be defined above its usage spot.